### DISCUSSION TITLE
Apple M1 GPU support on JAX?

### DISCUSSION THREAD

**Discussion by fbarfi** [jax-ml/jax#9847] [Q&A] [answered]
Just wondering if JAX might create support for Apple M1 GPU cores? So far Jax has worked fine with M1 CPU cores on MacBookPro M1 max. thanks.

**Comment by mattjj** [comment]
I think the discussion about this is at #8074. Does that sound right to you?

**Comment by fbarfi** [comment]
Thank you much. I followed the discussion thread and it looks promising, especially with the shark stuff on PyTorch.

**Comment by adam-hartshorne** [comment, marked as answer]
Looks like it is now available.

https://developer.apple.com/metal/jax/
